src/main/java/edu/kis/powp/jobs2d/drivers/composite/IDriverComposite.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.jobs2d.drivers.composite;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+
+
+public interface IDriverComposite extends Job2dDriver {
+
+
+    @Override
```

Comment:
Wrong formatting convention. Project uses tabs (not spaces).

Reply:
Switched from spaces to tabs.